config/credentials.default.coffee

```diff
@@ -1,3 +1,5 @@
+secret = () -> Math.random().toString(36).slice(-16)
```

Comment:
the rationale for this is that this secrets should be different for different open-source users, so one user won't be able to access different user kloud / terraformer services with default configuration - so it's safe by default; if someone wanted to have it fixed - one would provide his own config

wdyt?